app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    com, source, destination = command.split()
+
+    if com != "mv":
+        return
+
+    path_to_move = destination.split("/")
+
+    if len(path_to_move) == 1:
+        if os.path.exists(destination):
```

Comment:
The code does not handle the case where a directory already exists, which can result in errors. Proper error handling is needed to manage such scenarios gracefully.